.circleci/config.yml

```diff
@@ -346,7 +303,7 @@ jobs:
 # used in `build-docker-images` workflow below
 branch_filters: &branch_filters
   branches:
-    only: /.*deploy.*/
+    only: /cd\/staging/
```

Comment:
We are reusing prior behavior. This causes us to detect changes on `cd/staging` and trigger [this job](https://github.com/filecoin-project/lily/pull/1090/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R334).